### PR TITLE
Automatically add/remove labels when github review (requests) are used

### PR DESCRIPTION
**Pull request by oli-obk**

### CODE REVIEW

File: triagebot.toml [hunk 1]
```diff
@@ -17,3 +17,15 @@ contributing_url = "https://github.com/rust-lang/miri/blob/master/CONTRIBUTING.m
 
 [no-merges]
 exclude_titles = ["Rustup"]
+
+[review-submitted]
+# This label is added when a "request changes" review is submitted.
+reviewed_label = "S-waiting-on-author"
+# These labels are removed when a "request changes" review is submitted.
+review_labels = ["S-waiting-on-review"]
```
Comment (RalfJung) [inline triagebot.toml:25]: Does it matter who writes this review? Seems a bit odd to change the label if some random bystander leaves a review.

**Review by RalfJung — COMMENTED** [review]
Ah wow, I had no idea triagebot can do this. :D

File: triagebot.toml (hunk 1 above)
Comment (oli-obk) [reply triagebot.toml:25]: They cannot. Github only allows the request changes button if you have enough permissions on a repo

Comment (RalfJung) [reply triagebot.toml:25]: Ah, okay :)

**Pull request merged**